package.json

```diff
@@ -9,24 +9,24 @@
     "lib"
   ],
   "dependencies": {
-    "babel-plugin-syntax-object-rest-spread": "^6.13.0",
-    "find-up": "^2.1.0",
-    "istanbul-lib-instrument": "^1.10.1",
-    "test-exclude": "^4.2.1"
+    "@babel/plugin-syntax-object-rest-spread": "7.0.0-beta.51",
```

Comment:
We should probably introduce a `package-lock.json` since that seems to be our preference :)

Reply:
I've added this.